src/ImageGallery.js

```diff
@@ -49,6 +51,7 @@ class ImageGallery extends React.Component {
       isFullscreen: false,
       isSwipingThumbnail: false,
       isPlaying: false,
+      slideVertically: props.slideVertically,
```

Comment:
`slideVertically ` looks like it can just be a prop and does not need to be in `state` as its never changed.

Reply:
Got it, thanks for the feedback. I'll have time to finalize and refactor within the next two weeks 👍